src/librustc/mir/mod.rs

```diff
@@ -1520,6 +1520,9 @@ pub enum AggregateKind<'tcx> {
     /// active field number and is present only for union expressions
     /// -- e.g. for a union expression `SomeUnion { c: .. }`, the
     /// active field index would identity the field `c`
+    ///
+    /// For enums, the second field is the index of the variant
+    /// within AdtDef::fields
```

Comment:
Isn't this also 5 lines above?

Reply:
The confusion was "is it 0-indexed variant index, 1-indexed variant index, or the discriminant value". Basically, the fact that it says that it's zero for structs and unions suggests that it's perhaps nonzero for enums, and it's better if folks don't need to verify this.

Reply:
You can just edit that part instead of adding more elsewhere... also you wrote `AdtDef::fields` instead of `variants`. And this shouldn't be in this PR anyway.

Reply:
Actually structs and unions have exactly one variant so the rephrasing can be even more enum-focused.